server/webdriver_tests/tests/map_test.py

```diff
@@ -108,7 +106,7 @@ def test_charts_from_url(self):
         self.assertEqual(chart_title.text, "Median Age (2020)")
         chart_map = self.driver.find_element_by_id('map-items')
         map_regions = chart_map.find_elements_by_tag_name('path')
-        self.assertEqual(len(map_regions), 52)
+        self.assertEqual(len(map_regions), 58)
```

Comment:
This is mysterious to me. 58 is the right answer but somehow before it only sees 52.

Reply:
hm, interesting ..

Reply:
This is actually a bug. 58 is only for initial page load

Reply:
So the map works fine here. But now the chart does not unmount and mount, so all the html elements are not destroyed. The test sees an old value from laster rendering before the new rendering kicks in. 

I have a pending PR to add spinner. With that, can revisit the logic here to additional check spinner not shown.